### ISSUE TITLE
Hyperparameter Tuning with Ray Tune

### ISSUE THREAD

**Issue by HornGate** [ultralytics/ultralytics#2169]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

**fl_gamma** should be removed in default search space description when we do **Hyperparameter Tuning with Ray Tune**.

**_fl_gamma | tune.uniform(0.0, 2.0) | Focal loss gamma_**

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@AyushExel the fl_gamma param has been removed from default.yaml, it seems like it's still used in tuning?

**Comment by HornGate** [comment]
> @AyushExel the fl_gamma param has been removed from default.yaml, it seems like it's still used in tuning?

Yes. I think it's necessary to edit the tune code and the docs.

**Comment by AyushExel** [comment]
Oh yeah, it could be as I transferred the search space from v5 except the anchors.

**Comment by HornGate** [comment]
> Oh yeah, it could be as I transferred the search space from v5 except the anchors.
Thank you. Is it very slowly when we train with tune ray? It costs much more time even I train yolov8n detection model only with learning rate search space.

**Comment by HornGate** [comment]
> > Oh yeah, it could be as I transferred the search space from v5 except the anchors.
> > Thank you. Is it very slowly when we train with tune ray? It costs much more time even I train yolov8n detection model only with learning rate search space.



> Oh yeah, it could be as I transferred the search space from v5 except the anchors.

Thank you. Is it very slowly when we train with tune ray? It costs much more time even I train yolov8n detection model only with learning rate search space.

**Comment by glenn-jocher** [comment]
@HornGate training with Ray Tune can have longer training times due to the fact that it is used to search through a large space of hyperparameters. Additionally, the extent of the hyperparameters searched by Tune is generally greater than that used in a regular training run, hence the longer runtimes. However, it is important to note that hyper-tuning search spaces that are too large can result in over-fitting and longer run times.
